repos/system_upgrade/el7toel8/libraries/config/tests/test_version.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from leapp.libraries.common.config import version
+
+
+def test_version_to_tuple():
+    assert version._version_to_tuple('7.6') == (7, 6)
+
+
+def test_validate_versions():
+    assert version._validate_versions(['7.6', '7.7']) is None
```

Comment:
There's no sense in checking a return value, that's actually meaningless. You either get the exception or not. Here do just expect no exception to occur. If it would fail with an exception the test fails.
```suggestion
    version._validate_versions(['7.6', '7.7'])
```

Reply:
fixed